accounts-db/benches/bench_accounts_file.rs

```diff
@@ -34,7 +34,7 @@ fn bench_write_accounts_file(c: &mut Criterion) {
         let accounts: Vec<_> = std::iter::repeat_with(|| {
             (
                 Pubkey::new_unique(),
-                Account::new_rent_epoch(
+                AccountSharedData::new_rent_epoch(
```

Comment:
I did not implement `AccountForStorage` for `Account`. Just `AccountSharedData`. There are only a handful of test, bench, load from genesis code paths that need `Account`, so I just changed those.